front-end/tests/components/LoanList.test.js

```diff
@@ -31,17 +43,31 @@ describe('<LoanList />', () => {
   })
 
   it('<LoanList /> has 2 child', () => {
-    expect(component.toJSON().children.length).toBe(2)
+    expect(component.toJSON().children.length).toBe(1)
   })
 
-  it('<LoanList /> should dispatch an action when mounted', () => {
+  it('<LoanList /> should dispatch an action when mounted', async () => {
     renderer.act(() => {
       component = renderer.create(
         <Provider store={store}>
           <LoanList />
         </Provider>
       )
     })
-    expect(store.dispatch).toBeCalledWith(expectedPayload)
+
+    // eslint-disable-next-line no-undef
+    window.fetch = jest.fn().mockImplementation(() =>
```

Comment:
@JialinWang1 then why the need for `fetch-mock` library

Reply:
forgot to remove it